lib/vmpooler/prometheus.rb

```diff
@@ -0,0 +1,137 @@
+# frozen_string_literal: true
+
+require 'rubygems' unless defined?(Gem)
+
+# NB Check these 
+require 'prometheus/middleware/collector'
+require 'prometheus/middleware/exporter'
```

Comment:
Where are you thinking of adding these corresponding lines?

```ruby
use Prometheus::Middleware::Collector
use Prometheus::Middleware::Exporter
```

Reply:
This is TBD - need to re-circle back and address the Rack handling

Reply:
I'm guessing they could just live in the Class below? I don't think we'll get a `/metrics` endpoint without them

Reply:
I think I'm going to have to insert them twice - both in the api and the manager, and the manager needs to be converted to a Rack application also so that the endpoint is exposed.

Reply:
They should be in the top level rack / sinatra bits wrapped in a conditional that only actually uses them when prometheus is enabled IMO. We talked about a top level bit of Sinatra that would then include the bit desired (api, manger, or both for an AIO setup).